### PR TITLE
Feature api - Hit the Api

### PR DESCRIPTION
**Pull request by Anvi98**
- Read the Leaderboard API documentation to learn how to use this API.
- Create a new game with the name of your choice by using the API.
- Implement the "Refresh" button (receiving data from the API and parsing the JSON).
- Implement the form "Submit" button (sending data to the API).
- Use async and await JavaScript features to consume the API.
- No styling is required.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @Anvi98,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [x] After adding a score the score isn't shown in the list of scores. Please fix it.
- [x] Also, change the regular function expression to arrow function.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/apiCalls.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        div.classList.add('message-hidden');
+      }, 2000);
+    })
+    .catch((err) => err);
+};
+
+// Get info From Leadboard API
+export const callApi = (url) => {
+  board.get(url)
+    .then((res) => res.result)
```
Comment (mikenath223) [inline src/apiCalls.js:27]: - [x] After adding a score and clicking the refresh button, the score isn't shown in the list of scores. Please fix it. Check project [the requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md#:~:text=Implement%20the%20%22Refresh,to%20the%20API).

File: src/apiCalls.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    .then((res) => {
+      const form = document.querySelector('form');
+      const message = document.createElement('div');
+      message.classList.add('message-ok');
+      message.innerHTML = `${res.result}`;
+      form.appendChild(message);
+
+      setTimeout(() => {
+        message.classList.add('message-hidden');
+      }, 2000);
```
Comment (mikenath223) [inline src/apiCalls.js]: - [x] Please change this to es6 arrow function syntax. As stated in the project requirement [particularly here](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md#:~:text=If%20you%20use%20ES6%2C%20use%20object%20destructuring%20to%20get%20the%20values%20from%20an%20object.%20This%20way%20you%20can%20avoid%20repeating%20a%20lot%20of%20code.) in the JavaScript best practices section, it's important you use es6 syntax.

**Review by sinansevgi — APPROVED** [review]
# Approved :heavy_check_mark: 

Hi @Anvi98 , 

Great work with refactoring all of your functions with arrow syntax :+1: 
All the changes requested by the previous TSE were made.
Your project is complete!  
Congratulations! 🎉

You can merge this pull request and continue with your journey :wave: 


Happy Hacking! :computer: 

-----

You can reply to this conversation if you have any questions or need elaboration. :wave: 
Sinan Sevgi


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**